docs/elasticsearch-spec.asciidoc

```diff
@@ -116,6 +157,41 @@ spec:
         - dns: hulk.example.com
 ----
 
+[id="{p}-advanced-settings"]
+== Advanced Settings
+
+[id="{p}-virtual-memory"]
```

Comment:
There is another `virtual-memory` anchor on this page on L289 that breaks the docs build for me on this PR

Reply:
```
INFO:build_docs:/out/html_docs/index.xml:1343: element section: validity error : ID k8s-virtual-memory already defined
INFO:build_docs:<section id="k8s-virtual-memory">
```
@artemnikitin any idea why this not flagged up in CI?

Reply:
@pebrc based on CI logs everything was fine

Reply:
```
[2019-07-23T21:54:33.636Z] INFO:docker build:Successfully tagged docker.elastic.co/docs/build:1
[2019-07-23T21:54:33.636Z] INFO:build_docs:Building HTML from /doc/cloud-on-k8s/docs/index.asciidoc
[2019-07-23T21:54:38.927Z] INFO:build_docs:Done
```

Reply:
What I am trying to say here is that it should fail. I raised https://github.com/elastic/cloud-on-k8s/issues/1392